@stellar/typescript-wallet-sdk/examples/sep10/sep10Server.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
+
+// @ts-expect-error: express will need to be installed to run this example
```

Comment:
I thought it was maybe overkill to add `express` to the repo just for this example, so I just added this as a comment. Let me know if it's worth adding it so a user can actually spin this up from this repo

Reply:
Is it too heavy? I'm personally fine adding it as a dev dependency as we could use it for other examples as well, but also OK in case you think that's too much

Reply:
I guess it won't affect the bundle size of the library, either. I'll add this as a dev dep 👍